src/test/java/com/boardgo/integration/user/service/UserQueryServiceV1Test.java

```diff
@@ -0,0 +1,79 @@
+package com.boardgo.integration.user.service;
+
+import com.boardgo.common.exception.AlreadyElementExistException;
+import com.boardgo.domain.oauth2.entity.ProviderType;
+import com.boardgo.domain.user.controller.dto.EmailRequest;
+import com.boardgo.domain.user.controller.dto.NickNameRequest;
+import com.boardgo.domain.user.entity.UserInfoEntity;
+import com.boardgo.domain.user.repository.UserRepository;
+import com.boardgo.domain.user.service.UserQueryUseCase;
+import com.boardgo.integration.support.IntegrationTestSupport;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+public class UserQueryServiceV1Test extends IntegrationTestSupport {
+    @Autowired private UserRepository userRepository;
+    @Autowired private UserQueryUseCase userQueryUseCase;
+
+    @Test
+    @DisplayName("해당 이메일이 존재하지 않으면 아무일도 일어나지 않는다")
```

Comment:
별건아니지만.. 아래 닉네임 검증 처럼 구체적으로 일어나는 상황을 제시해도 좋을 것 같아요 ㅎㅎ
```suggestion
    @DisplayName("해당 이메일이 존재하지 않으면 에러가 발생하지 않는다")
```